2-mandatory/4-restaurant.js

```diff
@@ -20,8 +20,16 @@ const MENU = {
 };
 
 let cashRegister = {
-  // write code here
-}
+  orderBurger: function (balance) {
+    if (balance >= MENU.burger) balance = balance - MENU.burger;
+    return balance;
+  },
+  orderFalafel: function (balance) {
+    if (balance >= MENU.falafel) balance = balance - MENU.falafel;
+    return balance;
+  },
+};
+
```

Comment:
well done Shimen on completing all exercises & having passing tests- you've shown a good understanding of objects & accessing keys and values